src/App/App.js

```diff
@@ -14,10 +14,36 @@ class App extends Component {
         console.log(this.state.movies)
     }
 
+    componentDidMount = () => {
+        fetch("https://rancid-tomatillos.herokuapp.com/api/v2//movies/")
+        .then(res => {
+           if(!res.ok) {
+               throw new Error('Failed to fetch movies')
+           }
+           return res.json()})
+        .then(moviesData => this.setState({
+           movies: moviesData.movies
+        }))
+        .catch(error => {
+           console.error("There was a problem with the fetch", error)
+        })
+    }
+
     getMovieById = (id) => {
         const filteredMovie = this.state.movies.find(movie => movie.id === id)
-        this.setState({movie: filteredMovie})
-        console.log("filteredMovies", filteredMovie)
+
+        fetch(`https://rancid-tomatillos.herokuapp.com/api/v2//movies/${filteredMovie.id}/`)
+        .then(res => {
+           if(!res.ok) {
+               throw new Error('Failed to fetch the movie')
+           }
+           return res.json()})
+        .then(filteredMovieData => this.setState({
+           movie: filteredMovieData
```

Comment:
I looked up at the initial state for the 'movies' property to see what shape (data type) it would be holding.  I see the initial value of that property of state is 'null'.  Consider if there might be benefits, and what those benefits might be, to using an empty version of the expected data type instead of null.